src/earn/EarnApyAndAmount.tsx

```diff
@@ -0,0 +1,82 @@
+import BigNumber from 'bignumber.js'
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { StyleSheet, Text, View } from 'react-native'
+import { Colors } from 'react-native/Libraries/NewAppScreen'
+import { useSelector } from 'react-redux'
+import TokenIcon, { IconSize } from 'src/components/TokenIcon'
+import { useAavePoolInfo } from 'src/earn/hooks'
+import { LocalCurrencySymbol } from 'src/localCurrency/consts'
+import { getLocalCurrencySymbol } from 'src/localCurrency/selectors'
+import { typeScale } from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+import { TokenBalance } from 'src/tokens/slice'
+
+export function EarnApyAndAmount({
+  tokenAmount,
+  token,
+  testID,
```

Comment:
add a default. Otherwise, if one is not set, I think you'd get `undefined/EarnApyAndAmount/...`